app/src/main/java/com/android/unio/model/utils/Utils.kt

```diff
@@ -4,10 +4,32 @@ import android.content.Context
 import android.net.ConnectivityManager
 import androidx.core.content.ContextCompat.getSystemService
 
+enum class TextLength(val length: Int) {
+  SMALL(30),
+  MEDIUM(100),
+  LARGE(300)
+}
+
 object Utils {
 
   fun checkInternetConnection(context: Context): Boolean {
     val connectivityManager = getSystemService(context, ConnectivityManager::class.java)
     return connectivityManager?.activeNetwork != null
   }
+
+  fun checkInputLength(input: String, textLength: TextLength): Boolean {
+    return when (textLength) {
+      TextLength.SMALL -> input.length <= textLength.length
+      TextLength.MEDIUM -> input.length <= textLength.length
+      TextLength.LARGE -> input.length <= textLength.length
+    }
+  }
```

Comment:
This is just a tip for you, but it's always useful to use Kotlin patterns that make the code more readable. For example you could rewrite this function as an extension method of `String`, which could then be used like so:
```kotlin
"Some string".hasSize(SMALL)
```
Which is somewhat more readable than the current
```kotlin
checkInputLength("Some string", SMALL)
```
This is of course subjective and up to you to include or not.

Reply:
Wow okay I didn't even know about this, I'll look into it just by curiosity, thanks !